packages/contracts-bedrock/src/L1/SystemConfigInterop.sol

```diff
@@ -14,6 +14,51 @@ import { StaticConfig } from "src/libraries/StaticConfig.sol";
 ///         All configuration is stored on L1 and picked up by L2 as part of the derviation of
 ///         the L2 chain.
 contract SystemConfigInterop is SystemConfig {
+    /// @notice Storage slot where the dependency manager address is stored
+    bytes32 internal constant DEPENDENCY_MANAGER_SLOT =
+        bytes32(uint256(keccak256("systemconfig.dependencymanager")) - 1);
```

Comment:
wondering if there's a reason to not set it as?
```suggestion
        0x1708e077affb93e89be2665fb0fb72581be66f84dc00d25fed755ae911905b1c;
```

Reply:
It communicates the preimage/"how the value" is derived so the question of "why that particular storage slot?" isn't on the readers mind

Reply:
yes, but is there a defined way? in this other PR it's directly the value https://github.com/ethereum-optimism/optimism/pull/11398/files#diff-c899f14eedc7928371b7e497dd12658c2452d9a171856b2b001cc20885fe6dc8R46

Reply:
Oh, thought this was from the specs repo, yeah it can be hardcoded directly with a comment explaining the preimage

Reply:
Is there a standard approach though? Which is preferred or recommended?

Reply:
I think that using the exact `bytes32` value rather than the expression is preferred because its a gas optimization

Reply:
yeah thats true, though its used throughout the SystemConfig for 9 constants, need to remember to use the bytes32 values for that too...